app/lib/shared/utils.dart

```diff
@@ -158,6 +173,12 @@ void validatePackageName(String name) {
   if (_reservedWords.contains(name.toLowerCase())) {
     throw new Exception('Package name must not be a reserved word in Dart.');
   }
+  final bool isLower = name == name.toLowerCase();
+  final bool matchesMixedCase =
+      knownMixedCasePackages.any((s) => s.toLowerCase() == name.toLowerCase());
+  if (!isLower && !matchesMixedCase) {
```

Comment:
Why can't this be simply `if (!knownMixedCasePackages.contains(name)) { ... }` ?